decred/decred/dcr/rpc.py

```diff
@@ -275,25 +275,25 @@ def decodeScript(self, script, version=0):
 
     def estimateFee(self):
         """
-        Returns the estimated fee in dcr/kb.
+        Returns the estimated fee in atoms/kb.
 
         Returns:
-            float: Estimated fee.
+            int: Estimated fee in atoms/kb.
```

Comment:
I'll note that I would prefer atoms/B, but atoms/kB is the unit used for the `DefaultRelayFeePerKb` constant in both TD and dcrwallet, so I think we should standardize on that.